### PR TITLE
Create doc for hasura app

### PR DESCRIPTION
**Pull request by hitarth-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/webhooks/hasura.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+id: hasura
+title: Hasura
+description: Learn about the collection process for the Sumo Logic Hasura integration.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/webhooks/hasura-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Hasura Cloud is a fully-managed version of GraphQL Engine that gives you a scalable, globally-distributed, secure GraphQL API over your data sources. You can use a webhook in the Hasura platform to forward logs to the Sumo Logic HTTP endpoint. Using these logs, you can monitor the insert, update and deletion events in Sumo Logic. For more details, refer to the [Hasura Documentation](https://hasura.io/docs/latest/hasura-cloud/overview/).
```
Comment (JV0812) [inline docs/integrations/webhooks/hasura.md:10]:
```suggestion
Hasura Cloud is a fully-managed version of GraphQL Engine that provides you a scalable, globally-distributed, and secure GraphQL API over your data sources. You can use a webhook in the Hasura platform to forward logs to the Sumo Logic HTTP endpoint. Using these logs, you can monitor the insert, update, and deletion events in Sumo Logic. For more details, refer to the [Hasura Documentation](https://hasura.io/docs/latest/hasura-cloud/overview/).
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fix

File: docs/integrations/webhooks/hasura.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+id: hasura
+title: Hasura
+description: Learn about the collection process for the Sumo Logic Hasura integration.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/webhooks/hasura-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Hasura Cloud is a fully-managed version of GraphQL Engine that provides you a scalable, globally-distributed, and secure GraphQL API over your data sources. You can use a webhook in the Hasura platform to forward logs to the Sumo Logic HTTP endpoint. Using these logs, you can monitor the insert, update, and deletion events in Sumo Logic. For more details, refer to the [Hasura Documentation](https://hasura.io/docs/latest/hasura-cloud/overview/).
```
Comment (kimsauce) [inline docs/integrations/webhooks/hasura.md:10]:
```suggestion
Hasura Cloud is a fully managed version of GraphQL Engine that provides you with a scalable, secure, and globally distributed GraphQL API over your data sources. You can use a webhook in the Hasura platform to forward logs to the Sumo Logic HTTP endpoint. Using these logs, you can monitor the insert, update, and deletion events in Sumo Logic. For more details, refer to the [Hasura Documentation](https://hasura.io/docs/latest/hasura-cloud/overview/).
```

**Review by JV0812 — APPROVED** [review]
LGTM

**Pull request merged**